resources/assets/lib/interfaces/message-json.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import UserJson from 'interfaces/user-json';
+
+export default interface MessageJson {
```

Comment:
seems better either inside `interfaces/chat/` directory or prefixed with Chat (also ChannelJson and maybe ChatUpdatesJson)